core/src/subgraph/instance_manager.rs

```diff
@@ -42,6 +44,14 @@ lazy_static! {
     // Used for testing Graph Node itself.
     pub static ref DISABLE_FAIL_FAST: bool =
         std::env::var("GRAPH_DISABLE_FAIL_FAST").is_ok();
+
+    /// Ceiling for the backoff retry of non-deterministic errors, in milliseconds.
+    pub static ref BACKOFF_RETRY_CEILING_FOR_NON_DETERMINISTIC_ERRORS: Duration =
```

Comment:
@leoyvens also if you have any suggestion that's smaller than this one, I would love to hear it 😅

Reply:
Maybe `SUBGRAPH_ERROR_RETRY_CEIL`, and the env var should reflect the unit. I don't think millisecond is a useful granularity, I'd prefer seconds so `MAX_SUBGRAPH_ERROR_RETRY__CEIL_SECS`.